src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java

```diff
@@ -74,6 +69,14 @@ public String getUsage() {
         return "`{%}seek <time>` - Seeks to a specific time in the currently playing video.";
     }
 
+    @Override
+    public String getExtraInfo() {
+        return "**Time formatting**\n" +
+                "`h:m:s` - Skip to a specific amount of hours, minutes and seconds\n" +
+                "`m:s` - Skip to a specific amount of minutes and seconds\n" +
+                "`s` - Skip to a specific amount of seconds";
```

Comment:
Also mention the fact we support `1s`, `1h30m` etc

Reply:
Will do tomorrow, cba to rn